vterebey/budgetcalculator/budget_calculator.py

```diff
@@ -0,0 +1,306 @@
+"""
+'Budget Calculator' program to track expenses by category and date.
+To use you need to run the program. Next you need to choose what you
+are going to do, you can:
+1. Enter or select an expense category and cost
+2. View spending statistics on a pie chart
+3. Display expenses by category
+(you can choose the suggested one or enter it yourself)
+4. Display expenses by month (you must enter the year and
+month in the format: YYYY-MM)
+5. Display expenses by category and month at the same time
+(see above how the category and month are set)
+All expenses are recorded in the file 'values.csv'
+The file contains the category,date and cost
+"""
+
+
+import csv
+from datetime import date
+import matplotlib.pyplot as plt
+
+
+# dict 'categories' to store categories and amount
+fieldnames = ["Category", "Date", "Amount"]
+categories = {}
+default_categories = {
+    1: "Grocery store",
+    2: "Clothing",
+    3: "Entertainment",
+    4: "Cafe",
+    5: "Store",
+    6: "Pharmacy",
+}
+with open("values.csv", "w", encoding="UTF-8", newline="") as file:
+    writer = csv.writer(file)
+    writer.writerow(fieldnames)
+
+
+def add_category(category, amount):
+    """
+    adds category and it's amount to the dictionary
```

Comment:
1. **Minor:** String comment looks better, if it's capitalized :) Update for further comments too
2. Why don't we just add category but its value too if method' name is "add_category"?

Reply:
Comments will be corrected. Function names will say what the function does.